concert/devices/attenuators/base.py

```diff
@@ -0,0 +1,38 @@
+'''
+Attenuator module.
+Getter and Setter for filter material and the
+current state must be provided by the device implementation.
+'''
+
+from concert.base import State, check, AccessorNotImplementedError
+from concert.base import Parameter
+from concert.devices.base import Device
+
+
+class Attenuator(Device):
+    '''Attenuator base class'''
+    state = State(default='standby')
+    filter = Parameter(help='Filter material')
```

Comment:
First, `filter` is a keyword in Python, so the name must be different. Second, these are discrete choices, so I would use [Selection](https://github.com/ufo-kit/concert/blob/master/concert/base.py#L388-L406).